pkg/runner/job_runner.go

```diff
@@ -212,6 +203,21 @@ func (jr *JobRunner) Run(ctx xcontext.Context, j *job.Job, resumeState *job.Paus
 				runErr = jr.emitTargetEvents(runCtx, testEventEmitter, targets, target.EventTargetAcquired)
 			}
 
+			// Check for pause during target acquisition.
+			select {
+			case <-ctx.Until(xcontext.ErrPaused):
```

Comment:
I guess it won't be used, but still: it is also possible to write it this way:
```go
if ctx.IsSignaledWith(xcontext.ErrPaused) {
     ...
}
```
instead of
```go
select {
case <-ctx.Until(xcontext.ErrPaused):
    ...
default:
}
```

Reply:
ah, that's good to know, thanks